src/components/AssetManagementPanel/AssetManagementForm/FormVariants/HandleSurrenderedForm/HandleSurrenderedForm.tsx

```diff
@@ -5,43 +5,78 @@ import { LoaderSpinner } from "../../../../UI/Loader";
 import { AssetInformationPanel } from "../../../AssetInformationPanel";
 import { AssetManagementActions } from "../../../AssetManagementActions";
 import { AssetManagementTitle } from "../../AssetManagementTitle";
-import { OverlayContext } from "./../../../../../common/contexts/OverlayContext";
+import { OverlayContext } from "../../../../../common/contexts/OverlayContext";
 import {
   MessageTitle,
   showDocumentTransferMessage,
-} from "./../../../../../components/UI/Overlay/OverlayContent/DocumentTransferMessage";
+} from "../../../../UI/Overlay/OverlayContent/DocumentTransferMessage";
 import { TagBorderedRedLarge } from "../../../../UI/Tag";
+import { useEndorsementChain } from "../../../../../common/hooks/useEndorsementChain";
+import { TitleEscrowEvent } from "../../../../../types";
 
-interface AcceptSurrenderFormProps {
+interface HandleSurrenderedFormProps {
+  tokenId: string;
   formAction: AssetManagementActions;
   tokenRegistryAddress: string;
   beneficiary?: string;
   holder?: string;
   handleAcceptSurrender: () => void;
-  acceptSurrenderingState: string;
+  destroyingTokenState: string;
   setFormActionNone: () => void;
   setShowEndorsementChain: (payload: boolean) => void;
+  handleRejectSurrender: (lastBeneficiary: string) => void;
```

Comment:
restore?

Reply:
Also it assumes that future variation of the function will never be call with both bene and holder, will prefer to have both in the function signature from the start, and decided here when we pass in the values.